lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -82,7 +80,7 @@ void amf_ue_context_update_coll_keys(
         (const hash_key_t) ue_context_p->gnb_ngap_id_key);
     h_rc = hashtable_uint64_ts_insert(
         amf_ue_context_p->gnb_ue_ngap_id_ue_context_htbl,
-        (const hash_key_t) gnb_ngap_id_key, amf_ue_ngap_id);
+        (const hash_key_t) gnb_ngap_id_key, ue_context_p->amf_ue_ngap_id);
```

Comment:
why is this change needed? I think the contract of the `amf_ue_context_update_coll_keys` should be well defined.